.changeset/perfect-terms-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-amplify': patch
+---
+
+update readme
```

Comment:
Do we need a changeset?

Reply:
Upon further inspection, probably not because it looks like npm is pulling the readme from github rather than the tarball. however in this case it doesn't matter because `create-amplify` is already getting a minor bump in the next release